anyway/flask_app.py

```diff
@@ -1194,7 +1194,7 @@ def get(self):
     Returns infographics-data API
 """
 parser = reqparse.RequestParser()
-parser.add_argument("id", type=int, help="News flash id")
+parser.add_argument("news_flash_id", type=int, help="News flash id")
```

Comment:
@EliorGigi why did you perform this change? FE uses this API meaning we might need to modify things there